integration-tests/tests/run-catalog-workflow.js

```diff
@@ -10,23 +10,15 @@ const testCatalogFlowFn = _.flow(
 )(async ({ page, testUrl, token, workspaceName }) => {
   await enableDataCatalog(page, testUrl, token)
   await click(page, clickable({ textContains: 'browse & explore' }))
-  await waitForNoSpinners(page)
 
   await click(page, checkbox({ text: 'Granted', isDescendant: true }))
   await clickTableCell(page, 'dataset list', 2, 2)
-  await waitForNoSpinners(page)
-  await click(page, clickable({ textContains: 'Link to a workspace' }))
-  await waitForNoSpinners(page)
+  await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Link to a workspace' })) })
 
   await click(page, clickable({ textContains: 'Start with an existing workspace' }))
-  await click(page, input({ labelContains: 'Select a workspace' }))
-  await click(page, `//*[@role="combobox"][contains(normalize-space(.), "${workspaceName}")]`)
-  // The click call above is not actually selecting the workspace, so the Import button actually never becomes enabled.
-  // The "page.url().includes(workspaceName)" call returns false, but that doesn't actually fail the test.
-  // Instead of trying to check the URL, the test should check that the correct page loaded by looking for an expected element on the page.
-  // await click(page, clickable({ textContains: 'Import' }))
-  // await waitForNoSpinners(page)
-  // await page.url().includes(workspaceName)
+  await select(page, 'Select a workspace', `${workspaceName}`)
+  await noSpinnersAfter(page, { action: () => click(page, clickable({ text: 'Import' })) })
+  await findText(page, workspaceName)
```

Comment:
I would suggest looking for something a bit more specific than just the workspace name, because I would imagine that the workspace name can appear on a lot of pages. One possibility is the combination of the billing project and workspace name, which is what we show in the top context bar (though even that doesn't feel highly specific to the page to make sure loaded). `await findText(page, `${billingProject}/${workspaceName}`)`, which requires you to pass the `billingProject` into the test.

Is there any text or element on the page you expect to load that is definitely unique to that page?

Reply:
There isn't much else on the page:
<img width="1208" alt="Screen Shot 2022-02-10 at 4 09 49 PM" src="https://user-images.githubusercontent.com/6414394/153496884-16690dee-91b7-49b9-8b27-09aaf51c56ac.png">

Reply:
How about `await findText(page, ${billingProject}/${workspaceName})` and `await findText(page, 'Workspace Data')` since we expect to be in the data tab?

Reply:
The combination of `await findText(page, ${billingProject}/${workspaceName})` and looking or "Select a data type" would be good.